web/src/routes/(user)/photos/[[assetId=id]]/+page.svelte

```diff
@@ -32,33 +32,25 @@
 
   let { isViewing: showAssetViewer } = assetViewingStore;
   const assetStore = new AssetStore({ isArchived: false, withStacked: true, withPartners: true });
-  const assetInteractionStore = createAssetInteractionStore();
-  const { isMultiSelectState, selectedAssets } = assetInteractionStore;
+  const assetInteraction = new AssetInteraction();
 
-  let isAllFavorite = $state(false);
-  let isAllOwned = $state(false);
-  let isAssetStackSelected = $state(false);
-  let isLinkActionAvailable = $state(false);
-
-  $effect(() => {
-    const selection = [...$selectedAssets];
-    isAllOwned = selection.every((asset) => asset.ownerId === $user.id);
-    isAllFavorite = selection.every((asset) => asset.isFavorite);
-    isAssetStackSelected = selection.length === 1 && !!selection[0].stack;
-    const isLivePhoto = selection.length === 1 && !!selection[0].livePhotoVideoId;
+  let selectedAssets = $derived(assetInteraction.selectedAssetsArray);
```

Comment:
This variable is never written to or used. So, pretty sure we can just remove it completely

Reply:
It's not strictly necessary, but it improves the readability of the deriveds below that use this variable, so I’d prefer to keep it.